src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.ecommercee;
+
+public class UserController {
```

Comment:
co to tu robi? do tego w pakiecie com.kodilla.ecommercee

Reply:
nie wiem dlaczego mi to dorzuciło usunąłem w oddzielnym commicie